packages/modules/src/modules/epics/ContributionsEpic.tsx

```diff
@@ -117,23 +104,31 @@ const articleCountAboveContainerStyles = css`
     margin-bottom: ${space[4]}px;
 `;
 
-type HighlightedProps = {
-    highlightedText: string;
-    countryCode?: string;
-    numArticles: number;
-    tracking?: OphanTracking;
-    showAboveArticleCount: boolean;
-};
+// sendEpicViewEvent()
```

Comment:
I think I prefer moving stuff into separate files over comments like this

Reply:
I don't understand why the function is outside the component. I was tidying up the file and added the comment to highlight that it is - for unknown reasons - outside the component. If we do move the function to another file, do you have a suggestion about which file?

Reply:
Moved function into component